core/bounds.gms

```diff
@@ -288,6 +288,9 @@ loop(te$(sameas(te,"ngcc") OR sameas(te,"ngt") OR sameas(te,"gaschp")),
   vm_cap.lo("2020",regi,te,"1")$pm_histCap("2020",regi,te) = 0.95 * pm_histCap("2020",regi,te);
 );
 
+*** fix capacities for BECCS technologies to  zero in 2020
```

Comment:
I would suggest to be even more explicit here, as we technically constrain only the carbon capture part:
`fix capacities for advanced bio carbon capture technologies to zero in 2020 (carbon capture part of BECCS)`
What do you think?

Reply:
fair enough, I adjusted the text